src/Stratis.Features.FederatedPeg/Wallet/FederationWalletManager.cs

```diff
@@ -303,6 +303,27 @@ public void RemoveBlocks(ChainedHeader fork)
             }
         }
 
+        /// <summary>
+        /// Determines if the wallet is active.
+        /// The wallet will only become active after <see cref="FederationWallet.LastBlockSyncedHeight"/>.
+        /// </summary>
+        /// <param name="height">The height at which to test if the wallet should be active. Defaults to the chain indexer height.</param>
+        /// <returns></returns>
+        private bool IsWalletActive(int? height = null)
+        {
+            if (this.Wallet == null)
+                return false;
+
+            if (this.Wallet.LastBlockSyncedHash != null)
+                return true;
+
+            if (this.Wallet.LastBlockSyncedHeight >= (height ?? this.chainIndexer.Height))
```

Comment:
This means the wallet is head of consensus (should not happen but no harm I guess in checking)